core/storage/user/gae_models.py

```diff
@@ -1094,8 +1094,8 @@ def get_model_association_to_user():
 
     @classmethod
     def get_export_policy(cls):
-        """This model is not included because it contains data about other
-        users.
+        """Model contains data corresponding to a user, but this model is not
+        expotted because it contains data about other users.
```

Comment:
Nit (consistency): ``...because it contains data corresponding to other users.``

Reply:
Done